renderer/src/pages/Onboarding.tsx

```diff
@@ -20,22 +20,24 @@ const OnboardingPage = (): JSX.Element => {
   const navigate = useNavigate()
   const [isLoading, setIsLoading] = useState<boolean>(true)
   const [isOnboardingCompleted, setIsOnboardingCompleted] = useState<boolean|null>()
-
-  useEffect(() => { document.title = 'Filecoin Station' })
+  const [filAddress, setFilAddress] = useState<string|null>()
```

Comment:
interesting, why did this part of onboarding change?

Reply:
[bbd4874](https://github.com/filecoin-project/filecoin-station/pull/144/commits/bbd4874fb7d2c3334f0ba0cca3eaaba70f97b899)